### DISCUSSION TITLE
Procedura accesso da remoto con duckdns frizbox e sim vodafone

### DISCUSSION THREAD

**Discussion by Starc73** [indomus/forum#3142] [HUB: Home Assistant]
Salve ragazzi sono nuovo del settore e ricerco il vs aiuto per configurare l'accesso remoto di home assistant, ho un nuc con installato HASSIO e ho già fatto la procedura
https://indomus.it/guide/collegarsi-da-remoto-a-home-assistant-installato-con-distribuzione-hassio/
sto configurando il port forwarding sul router fritzbox 6820 con sim vodafone. Con whatismyipaddress ho recuperato il mio Ip4 collegato al fritz vodafone, sono entrato nel mio frizbox e in abilitazioni ho inserito l'IP e poi la porta interna ed esterna 8123 ma  il fritz mi dice "l'indirizzo ip non si trova in una rete permessa dal fritzbox". Cosa ho sbagliato? Grazie in anticipo

**Comment by KC-inDomus** [comment]
non ti funzionerà mai questa guida, è normale: se usi una sim per collegarti, hai bisogno di un tunnelling - ergo, questa guida: https://indomus.it/guide/collegarsi-da-remoto-a-home-assistant-via-reti-provider-tradizionali-o-che-usino-nat-via-cloudflare/
